packages/analytics/lib/modular/index.js

```diff
@@ -64,6 +65,20 @@ export function initializeAnalytics(app, options) {
   return getApp(app.name).analytics();
 }
 
+/**
+ * Retrieves a unique Google Analytics identifier for the web client.
+ *
+ * @param {FirebaseAnalytics} analytics - Instance of analytics (web - only)
+ * @returns {Promise<string>}
+ */
+export async function getGoogleAnalyticsClientId(analytics) {
+  if (Platform.android || Platform.ios) {
```

Comment:
Platform.other is defined in globals in our e2e test suite but in general usage it isn't defined well

https://github.com/invertase/react-native-firebase/blob/c26253c82f0a13f62608941d266a63071667cc16/tests/globals.js#L95-L102

When running on macOS Platform.other doesn't show up the way you expect
So I define it here basically the way we define in testing globals